### ISSUE TITLE
Accessibility Hover in Footer

### ISSUE THREAD

**Issue by yiwenxx** [pulibrary/aux#36]
### **Issue**

- The current Accessibility link in the footer has a standalone hover effect, which the entire text background changes to white and text color changes to black, not aligning to all other hover effects.

![Image](https://github.com/user-attachments/assets/0eb32672-c8b2-422a-8b8e-5bebb14682f9)

### **Implementation Notes**

- Keep the hover state the same as all other links. When hover, the content will be underlined.

![Image](https://github.com/user-attachments/assets/546aaf22-66a2-46d0-b66e-e7bee245efbb)

**Demo**

![Image](https://github.com/user-attachments/assets/ef0d0509-34b8-408f-8be9-250e09149f69)

Link to Figma: https://www.figma.com/design/rgzEYTpnGl6l68moRSVwYg/Phase-1-UI-Details?node-id=2536-4948&t=I3GXx7fYln7e6HP5-4
